### ISSUE TITLE
Installation problems

### ISSUE THREAD

**Issue by tschoonj** [mretegan/crispy#21]
Hi,

I want to install crispy in an anaconda environment according to your instructions but I am getting an exception when launching `crispy`:

```
Traceback (most recent call last):
  File "/usr/local/miniconda3/envs/crispy/bin/crispy", line 11, in main
    from crispy.__main__ import main
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/site-packages/crispy/__main__.py", line 37, in <module>
    from .gui.main import MainWindow
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/site-packages/crispy/gui/main.py", line 50, in <module>
    from .quanty import QuantyDockWidget, QuantyPreferencesDialog
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/site-packages/crispy/gui/quanty.py", line 402, in <module>
    class QuantyCalculation(object):
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/site-packages/crispy/gui/quanty.py", line 451, in QuantyCalculation
    path = resourceFileName('quanty:parameters/parameters.json.gz')
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/site-packages/silx/resources/__init__.py", line 251, in resource_filename
    return _resource_filename(resource, default_directory=None)
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/site-packages/silx/resources/__init__.py", line 288, in _resource_filename
    return pkg_resources.resource_filename(package_name, resource_name)
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1144, in resource_filename
    return get_provider(package_or_requirement).get_resource_filename(
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/site-packages/pkg_resources/__init__.py", line 364, in get_provider
    return _find_adapter(_provider_factories, loader)(module)
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1392, in __init__
    self.module_path = os.path.dirname(getattr(module, '__file__', ''))
  File "/usr/local/miniconda3/envs/crispy/lib/python3.7/posixpath.py", line 156, in dirname
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/miniconda3/envs/crispy/bin/crispy", line 24, in <module>
    main()
  File "/usr/local/miniconda3/envs/crispy/bin/crispy", line 20, in main
    main()
UnboundLocalError: local variable 'main' referenced before assignment
```

Any thoughts? Thanks!

**Comment by mretegan** [comment]
Unfortunately there are some missing packages in the 0.7.2 version. Until the next release you can install the current version using:
```pip install --upgrade --user https://github.com/mretegan/crispy/tarball/master```.

**Comment by mretegan** [comment]
Fixed for 0.7.3 release.

**Comment by tschoonj** [comment]
Thanks, I will give it a try and let you know if I run into problems.
